index.html

```diff
@@ -588,5 +588,431 @@ <h2>Exposing change of MediaStreamTrack configuration</h2>
     </p>
     </div>
   </section>
+  <section>
+    <h2>Face detection</h2>
+    <section>
+      <h3>{{VideoFrame}}</h3>
+      <pre class="idl"
+>partial interface VideoFrame {
+  readonly attribute FrozenArray&lt;DetectedFace&gt;? detectedFaces;
```

Comment:
Based on discussions in https://github.com/w3c/webcodecs/pull/559, the direction might be to move to
`partial dictionary VideoFrameMetadata`